assembl/static2/tests/unit/actions/adminActions.spec.js

```diff
@@ -0,0 +1,75 @@
+import * as actions from '../../../js/app/actions/adminActions';
```

Comment:
yeah, no problem with that as it "puts" all imported stuff in `actions` variable (you are less likely to get overridden variables )